internal/proxy/usecases/proxy_usecase.go

```diff
@@ -70,6 +73,22 @@ func (p *ProxyUsecase) findInDatabaseIfNotFoundInCache(UUID string) error {
 	return nil
 }
 
+// WriteCachedRequestsRoutine  write metric cache in database
+func (p *ProxyUsecase) WriteCachedRequestsRoutine() {
+	logrus.Info("func WriteCachedRequestsRoutine")
```

Comment:
I suggest a log like : "Starting to write cached requests to database"

Reply:
replaced